README.md

```diff
@@ -369,6 +371,20 @@ an instance of the row that is clicked. It also adds the bootstrap `table-hover`
 />
 ```
 
+#### Mouse Events
+
+For more complex interactions, such as supporting the ability to Middle-click, you can use the `onMouseUp` and `onMouseDown` events instead. It also adds the bootstrap `table-hover` class onto the table.
+
+```JSX
+<DynamicDataTable
+    rows={this.state.users}
+    onMouseDown={(event, row) => console.log("Pressing button: " + event.button + " on row: " + row.id)}
```

Comment:
Can we update these to match our internal standards? Templating rather than concatenation.